mesonbuild/compilers/cpp.py

```diff
@@ -393,15 +395,15 @@ def __init__(self, ccache: T.List[str], exelist: T.List[str], version: str, for_
 
     def get_options(self) -> 'MutableKeyedOptionDictType':
         opts = CPPCompiler.get_options(self)
+
+        key = self.form_compileropt_key('eh')
+        opts[key] = options.UserComboOption(
+            self.make_option_name(key),
+            'C++ exception handling type.',
```

Comment:
nitpick: this one ends with a `.`, while other options don't...

Reply:
That has apparently always existed. The main branch right now has that peroid